daml/Marketplace/Issuance/Instrument/Service.daml

```diff
@@ -27,6 +29,8 @@ template Service
           let cfi = CFI with code = "DBXXXX"
           claims <- getClaims operator provider bond.stream bond.currencyId bond.maturityDate True
           bondCid <- create bond
+          create ClaimToken with id = (bond.id with label = bond.claimTokenId); originalId = bond.id; observers = fromList [operator]
```

Comment:
Can we just pass an `Optional Text` here as the `claimTokenId` parameter, so that we don't have to store it on the `Bond` template, and we only create the claim token contracts when this is specified.

Reply:
Updated as suggested